src/components/search-query.jsx

```diff
@@ -96,22 +108,48 @@ HitComponent.propTypes = {
     office_title: PropTypes.string,
   }),
 };
-HitComponent.defaultProps = {
-  hit: {},
+
+Hits.defaultProps = {
+  hits: {},
 };
 
+const Results = connectStateResults(({ searchState }) =>
+  (searchState && searchState.query ? (
+    <div>
+      <CustomHits />
+    </div>
+  ) : (
+    <div>No query</div>
+  )));
+
 const SearchQuery = () => (
   <InstantSearch searchClient={searchClient} indexName="election">
     <div className="grid">
       <div className="grid-col-2 election-checkboxes">
         <ClearRefinements clearsQuery="true" />
         <h3>Election Title</h3>
-        <RefinementList attribute="election_title" />
+        <RefinementList
```

Comment:
Similar to the `Hits` component above, now that this component is now handling some custom logic, it is a prime candidate to be separated out into its own file.